src/lib/homestore_backend/resync_blob_data.fbs

```diff
@@ -2,18 +2,16 @@ native_include "sisl/utility/non_null_ptr.hpp";
 
 namespace homeobject;
 
-table BlobData {
-    shard_id : uint64;
+table ResyncBlobData {
     blob_id : uint64;
-    user_key_size : uint32;
-    data_size : uint32;
-    user_key : [ubyte];
-    data : [ubyte];
+    is_deleted: bool;
+    data : [ubyte];    // Raw blob data loaded from drive, include BlobHeader, user_key and payload
```

Comment:
where do we want to do checksum ?

Reply:
BlobHeader already has a hash https://github.com/eBay/HomeObject/blob/a90a20f79f13b3623df4c339cea2b82a0b035400/src/lib/homestore_backend/hs_homeobject.hpp#L250